src/stactools/ecmwf_forecast/kerchunk_helper_functions.py

```diff
@@ -0,0 +1,60 @@
+from kerchunk.combine import MultiZarrToZarr
```

Comment:
Need to ensure kerchunk, numcodecs, and fsspec are in the requirements in the pyproject.toml

Reply:
Done.